modules/ROOT/pages/6.0-release-notes-options.adoc

```diff
@@ -0,0 +1,68 @@
+= TinyMCE 6.0 Options changes
+:navtitle: TinyMCE 6.0 Options changes
+:description: TinyMCE 6.0 Options changes
+:keywords: releasenotes, options, configurations
+
+This section documents the changes made to {productname} 6.0 Options.
+
+It also documents the {productname} 6.0 Option that has been removed.
+
+* xref:options[Options]
+* xref:changed-options[Changed Options]
+* xref:removed-or-deprecated-apis[Removed or deprecated Options]
+
+// tag::options[]
+[[options]]
+== Options
+
+
+[[changed-options]]
+=== Changed options
+
+
+[[image-upload-handler]]
+==== `image_upload_handler`
+
+Previously the `images_upload_handler` option passed `success` or `failure` callbacks to report the status of an image upload.
+
+In {productname} 6, `images_upload_handler` reports upload results by returning a `Promise`, with a value of either `resolve` or `reject`.
```

Comment:
This is a general point, repeated multiple times in the doc, but your description is not how JavaScript promises work.

Promises can be created in two ways; either resolved with a value or rejected with an error. These are not themselves values of Promise.

To be honest, I think we can just remove the 'with a value' text completely. Anyone who doesn't already know how promises work isn't going to figure them out by us trying to explain that here.
```suggestion
In {productname} 6, `images_upload_handler` reports upload results by returning a `Promise`.
```

- [ ] if this is accepted repeat the change every other time it appears